articles/contributing/docs.md

```diff
@@ -46,42 +46,74 @@ When we compile each release of the Quantum Development Kit, these comments are
 > Please make sure to not manually edit the generated API documentation, as these files are overwritten with each new release.
 > We value your contribution to the community, and want to make sure that your changes continue to help users release after release.
 
-For example, consider an operation `PrepareTrialState(angles : Double[], register : Qubit[]) : Unit`.
-A documentation comment should help a user learn how to interpret `angles`, what the operation assumes about the initial state of `register`, what the effect on `register` is, and so forth.
+For example, consider the function `ControlledOnBitString<'T> (bits : Bool[], oracle : ('T => Unit is Adj + Ctl)) : ((Qubit[], 'T) => Unit is Adj + Ctl)`.
+A documentation comment should help a user learn how to interpret `bits` and `oracle` and what the function is for.
 Each of these different pieces of information can be provided to the Q# compiler by a specially named Markdown section in the documentation comment.
-For the example of `PrepareTrialState`, we might write something like the following:
+For the example of `ControlledOnBitString`, we might write something like the following:
 
 ```qsharp
-/// # Summary
-/// Given a register of qubits, prepares them in a trial state by rotating each
-/// independently.
-///
-/// # Description
-/// This operation prepares the input register by performing a
-/// $Y$ rotation on each qubit by an angle given in `angles`.
-///
-/// # Input
-/// ## angles
-/// An array of parameters
-/// ## register
-/// A register of qubits initially in the $\ket{00\cdots0}$ state.
-///
-/// # Example
-/// To prepare an equal superposition $\ket{++\cdots+}$ over all input qubits:
-/// ```qsharp
-/// PrepareTrialState(ConstantArray(Length(register), PI() / 2.0), register);
-/// ```
-///
-/// # Remarks
-/// This operation is generally useful in the inner loop of an optimization
-/// algorithm.
-///
-/// # See Also
-/// - Microsoft.Quantum.Intrinsic.Ry
-operation PrepareTrialState(angles : Double[], register : Qubit[]) : Unit {
-    // ...
-}
+ /// # Summary
+ /// Returns a unitary operator that applies an oracle on the target register if the control register state corresponds 
```

Comment:
It looks like some of the improvements to the documentation comment suggested by @bettinaheim are still on the feature/qml branch, sorry. Would you be willing to use https://github.com/microsoft/QuantumLibraries/blob/feature/qml/Standard/src/Canon/Utils/ControlledOnBitString.qs instead here?

Reply:
Ok! Done!